Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1812,7 +1813,11 @@ namespace Remotion.BocAutoCompleteReferenceValue
         const minWidth = LayoutUtility.GetOuterWidth(reference);
         const maxWidth = Math.min (isNaN (options.maxWidth) ? LayoutUtility.GetOuterWidth(reference) : options.maxWidth, availableWidth);
         const requiredWidth = contentWidth + 30;
-        const elementWidth = Math.max (Math.min (requiredWidth, maxWidth), minWidth);
+
+        let elementWidth: number | string;
+        elementWidth = Math.max (Math.min (requiredWidth, maxWidth), minWidth);
+        if(requiredHeight == maxHeightSafe)
+            elementWidth = "auto";
```

Comment:
Stylistic: I would prefer
```
if (...)
  elemWidth = ...
else
  elemWidth = ...
```
Reasoning, aside from symmetry: avoiding multiple assignments to the same variable whenever possible. Makes things more deterministic. 

Or, further refactoring, and considering the Javascript syntax needed / possibe:
`const elemWidth = reqHght == maxHeightSafe ? "auto" : ...;`

Sensible style change?